lib/styles/themes/base/def/colors.js

```diff
@@ -67,4 +67,24 @@ export default {
   TEXT_LIGHT: `
     color: var(--text-light);
   `,
+  TEXT_ERROR: `
+    color: var(--error);
+  `,
+  FIELD_DEFAULT: `
+    color: var(--text-cta-tertiary);
+    background-color: var(--bg-cta-tertiary);
+  `,
+  FIELD_DISABLED: `
+    background-color: var(--brand-disabled);
+    color: 1px solid var(--text-cta-disabled);
+  `,
+  GREY_COLOR: `
```

Comment:
@sahimabhan we don't create such generic variables instead we use them to create more contextual objects.